cs/cli/vw_arguments.h

```diff
@@ -36,11 +36,16 @@ public ref class VowpalWabbitArguments
   float m_power_t;
 
   internal : VowpalWabbitArguments(VW::workspace* vw)
```

Comment:
Does this mean C# cannot support multiple data files?

Reply:
yes, CLI for now and python thanks to the duck typing and support for lists (-q and other args)